kratos/containers/container_expression/expressions/expression.h

```diff
@@ -82,11 +162,21 @@ class KRATOS_API(KRATOS_CORE) Expression {
     ///@name Input and output
     ///@{
 
+    ExpressionIterator cbegin() const;
+
+    ExpressionIterator cend() const;
```

Comment:
```suggestion
    ExpressionIterator begin() const;

    ExpressionIterator end() const;

    ExpressionIterator cbegin() const;

    ExpressionIterator cend() const;
```

Does the same thing with standard `begin` and `end`.

Reply:
done :)